src/handler.php

```diff
@@ -31,7 +31,7 @@ public function cleanEvent(Event &$e) {
         $event->setLocation($location);
 
         //Remove the TAG and anything after e.g.: (IN0001) or [MA0001]
-        $summary = preg_replace('/([\(\[](?:(?:IN|MA|WI)\d+,?\s?)+[\)\]]).+/', '', $summary);
+        $summary = preg_replace('/([(\[](?:(?:IN|MA|WI)\d+,?\s?)+[)\]]).+/', '', $summary);
```

Comment:
Whats the reason behind this change?

Reply:
I do as Intellij guides :smile: 

It was marked as redundant, so I figured, why not update this as well?

Reply:
Fine by me :)